streamlinecms/core/settings.py

```
@@ -0,0 +1,81 @@
+    def database_uri(self):
+        project_settings = ProjectSettings()
+        if project_settings.test:
+            return f"postgresql://{self.user}:{self.password}@{self.host}:{self.port}/{self.database}_test"
+        return f"postgresql://{self.user}:{self.password}@{self.host}:{self.port}/{self.database}"
+
+    @property
+    def echo(self):
+        project_settings = ProjectSettings()
+        if project_settings.env == "prod":
+            return False
+        return True
```

Ruff

Return the condition `not project_settings.env == "prod"` directly

Replace with `return not project_settings.env == "prod"`

(SIM103)